cats-frontend/src/app/features/applications/application/applicationTabs/appParticipants/ParticipantsTable.tsx

```diff
@@ -108,6 +202,67 @@ const ParticipantTable: React.FC<IParticipantTableProps> = ({
             </Button>
           </div>
         )}
+        {appParticipant.isAppParticipantModal && (
+          <ModalDialog
+            headerLabel={appParticipant.appParticipantActionType}
+            saveButtonDisabled={
+              appParticipant.appParticipantDetails.fullName === '' ||
+              appParticipant.appParticipantDetails.description === '' ||
+              appParticipant.appParticipantDetails.effectiveStartDate === ''
+            }
+            closeHandler={(response) => {
+              if (response) {
+                if (
+                  appParticipant.appParticipantActionType ===
+                  AppParticipantsActionTypes.AddParticipant
+                ) {
+                  const newAppParticipant = {
+                    applicationId: id ? Number(id) : 0,
+                    isMainParticipant:
+                      appParticipant.appParticipantDetails.isMainParticipant,
+                    personId: parseFloat(
+                      appParticipant.appParticipantDetails.fullName,
+                    ),
+                    organizationId: parseFloat(
+                      appParticipant.appParticipantDetails.name,
+                    ),
+                    participantRoleId: parseFloat(
+                      appParticipant.appParticipantDetails.description,
+                    ),
+                    effectiveStartDate:
+                      appParticipant.appParticipantDetails.effectiveStartDate,
+                    effectiveEndDate:
+                      appParticipant.appParticipantDetails.effectiveEndDate ||
+                      null,
+                  };
+                  handleAddAppParticipant(newAppParticipant)
+                    .then((response) => {
+                      if (response?.data?.createAppParticipant.success) {
+                        handleRefreshParticipants();
+                      }
+                    })
+                    .catch((err) => {
+                      console.error('Error create note:', err);
+                    });
```

Comment:
I suggest replacing this call with this:

```suggestion
                  handleAddAppParticipant(newAppParticipant);
```


`handleAddAppParticipant` itself can now look like this:

```typescript
const handleAddAppParticipant = async (
    newParticipant: CreateAppParticipantDto,
  ) => {
    return createAppParticiant({
      variables: {
        newAppParticipant: newParticipant,
      },
      onCompleted: () => {
        handleRefreshParticipants();
      },
      onError: (error) => {
        console.error('Error adding participant:', error);
      },
    });
  };
```

This is not a blocker by any means, I just think it reads a bit better and we're coupling the success/error management to the mutation call, not the data it returns.

Reply:
Nice!